Task1/intro.json

```diff
@@ -324,21 +324,9 @@
     "hacktoberfest_round": "first",
     "role": ["contributor"]
   },
-  {

```

Comment:
Hello @Iamthevk , Please be sure that you have not deleted any existing texts.
Can you kindly replace this text that have been deleted?